common/source/docs/common-fettec-onewire.rst

```diff
@@ -0,0 +1,56 @@
+==============
+FETtec OneWire
+==============
+
+This is a half duplex serial protocol with a 2Mbit/s Baudrate created by Felix Niessen (former Flyduino KISS developer) from `FETtec <https://fettec.net/>`.
+You can change many more settings of the ESC over OneWire than over DShot.
```

Comment:
This is advertisement statement that should be removed

Reply:
@teezeefpv can you provide proff of this? Or is it just mambo-jumbo?
Originaly I got this from https://blog.seidel-philipp.de/what-is-fettec-onewire/ because that page actualy has more information than your website. :(

Reply:
This bring nothing on ArduPilot, we should just remove it. And let people do their search on what protocol they think are the better.

Reply:
Done